wondrous-app/components/Settings/styles.tsx

```diff
@@ -273,6 +279,16 @@ export const SettingsHeaderInviteButtonIcon = styled.div`
   border-radius: 1000px;
 `;
 
+export const SettingsHeaderActionText = styled(ButtonBase)`
```

Comment:
it's same as `<Typography color={palette.highlightBlue} fontSize={14} letterSpacing="0.01em" lineHeight={18}>`, you don't need styled component

Reply:
I believe a button makes more sense here semantically as this when clicked downloads the org / dao members to a csv file. Let me know your thoughts on this